src/game.cpp

```diff
@@ -3853,25 +3851,39 @@ void Game::updateFrame(std::vector<aabb3f> &highlight_boxes,
 		Inventory
 	*/
 
-	if (client->getPlayerItem() != runData->new_playeritem)
+	bool inventory_selection_changed = false;
+	bool inventory_changed = false;
```

Comment:
est31 is correct.

The only objects that are initialized (to 0) are globals and static locals (these are not the terms the standard uses, but it's close enough)